python/ql/lib/semmle/python/security/dataflow/UrlRedirectCustomizations.qll

```diff
@@ -142,7 +142,7 @@ module UrlRedirect {
   /**
    * A comparison with a constant string, considered as a sanitizer-guard.
    */
-  class StringConstCompareAsSanitizerGuard extends Sanitizer, StringConstCompareBarrier {
+  class StringConstCompareAsSanitizerGuard extends Sanitizer, ConstCompareBarrier {
```

Comment:
Did you mean to follow the same pattern of deprecation here?

Reply:
Yes; I had missed it with a find-and-replace. 
Updated.